gateway/src/main/java/io/zeebe/gateway/EndpointManager.java

```diff
@@ -141,6 +144,26 @@ public void createWorkflowInstance(
         responseObserver);
   }
 
+  @Override
+  public void createWorkflowInstanceWithResult(
+      CreateWorkflowInstanceWithResultRequest request,
+      StreamObserver<CreateWorkflowInstanceWithResultResponse> responseObserver) {
+    if (request.getRequestTimeout() > 0) {
```

Comment:
What is the grpc default timeout?

Reply:
Thanks @Zelldon for fast review.
The default value for the field 'requestTimeout' in the grpc message is 0. If the value is 0, we use the same timeout that we use for other requests - default 15s, but configurable. I will make it explicit in the docs.